cvat-ui/src/components/annotation-page/top-bar/right-group.tsx

```diff
@@ -185,9 +185,19 @@ function RightGroup(props: Props): JSX.Element {
                                 </Select.Option>
                             );
                         }
+                        if (jobInstance.task.allowedAnnotationMode === AllowedInstruments.TAG) {
+                            if (ws === Workspace.STANDARD3D) {
+                                return null;
+                            }
+                            return (
+                                <Select.Option disabled={ws === Workspace.ATTRIBUTE_ANNOTATION} key={ws} value={ws}>
+                                    {ws}
+                                </Select.Option>
+                            );
+                        }
                         if (ws !== Workspace.STANDARD3D) {
                             return (
-                                <Select.Option key={ws} value={ws}>
+                                <Select.Option disabled={ws !== Workspace.STANDARD} key={ws} value={ws}>
```

Comment:
you are already in the if branch, where `ws !== Workspace.STANDARD3D`. this condition is always true here. just leave
```
<Select.Option disabled key={ws} value={ws}>
```

Reply:
Are you sure?
`if (ws !== Workspace.STANDARD3D)` means that there are 3 possible variants returning true.
And `disabled={ws !== Workspace.STANDARD}` makes them disabled except of STANDARD.

Reply:
ahh, ok. agree